src/Contracts/MASA.Scheduler.Contracts.Server/Model/Job.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace MASA.Scheduler.Contracts.Server.Model;
```

Comment:
Model? Dto?

Reply:
Already changed in branch feature/domainAggregates